.github/workflows/ci-core-reusable.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         base_token: ["Eth", "Custom"]
         deployment_mode: ["Rollup", "Validium"]
     env:
-      SERVER_COMPONENTS: "api,tree,eth,state_keeper,housekeeper,commitment_generator${{ matrix.consensus && ',consensus' || '' }}"
+      SERVER_COMPONENTS: "api,tree,eth,state_keeper,housekeeper,commitment_generator${{ matrix.consensus && ',consensus' || '' }}${{ matrix.deployment_mode == 'Validium' && ',da_dispatcher' || '' }}"
```

Comment:
I went with the more explicit approach, where it is required to set `pubdata_sending_mode="Custom"` to run the `da_dispatcher` to ensure that the component is not misconfigured, otherwise, it panics. But we can also go with just skipping the component's init in case `pubdata_sending_mode` is not `Custom`, but it is more an implicit logic, that I like less

If you have a preference towards launching it by 'default' - I can change it to be like that